src/leases/components/leaseSections/rent/ContractRents.tsx

```diff
@@ -121,6 +127,16 @@ class ContractRents extends PureComponent<Props, State> {
                     </>
                   </Authorization>
                 </Column>
+                {(serviceUnitId !== ServiceUnitIds.MAKE) && <Column small={6} medium={4} large={3}>
```

Comment:
Also here a comment could clarify why MAKE is not shown

Reply:
Added in the other PR in commit https://github.com/City-of-Helsinki/mvj-ui/pull/544/commits/494f4e63e087239105cb47bbd71561961bce3345